### PR TITLE
V6

### PR DESCRIPTION
**Pull request by nd2b**
v6 update

### CODE REVIEW

File: arg-gan.ipynb [hunk 1]
```diff
@@ -9,7 +9,6 @@
     "import pandas as pd\n",
```
Comment (somuchkindof2ch) [inline arg-gan.ipynb:9]: <p>Думаю, можно удалить.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/somuchkindof2ch/artgan/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (somuchkindof2ch) [inline arg-gan.ipynb:9]: <p>*img=cv2.imread(path_to_train_set + name)*</p><p>При объединений файловых путей во избежание ошибок лучше использовать <em>os.path.join </em>или <em>pathlib</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/somuchkindof2ch/artgan/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (somuchkindof2ch) [inline arg-gan.ipynb:9]: <p>Строки 33-35 - после ConvTranspose нужны функции активации и нормализация. Каждой.</p><p> </p><blockquote>self.deconv1 = nn.Sequential(*block(some_input, 128, 4, 2, 1))&nbsp;</blockquote><p>откуда взялась переменная some_input? Её в принципе нигде нет.</p><p> </p><blockquote>def block(in_channels, out_channels, kernel_size, stride, padding)</blockquote><p>не хватает двоеточия.</p><p> </p><blockquote>linear1 = F.LeakyReLU(self.linear1(n_features))</blockquote><p>почему Linear слой взят от <strong>числа</strong>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/somuchkindof2ch/artgan/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (somuchkindof2ch) [inline arg-gan.ipynb:9]: <p>Нигде далее не используется, удалить.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/somuchkindof2ch/artgan/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (somuchkindof2ch) [inline arg-gan.ipynb:9]: <blockquote>gen_loss_avg.append(0)</blockquote><blockquote>disc_loss_avg.append(0)</blockquote><p>Зачем первыми значениями лосса писать нули?</p><p> </p><p>F.binary_cross_entropy в disc_loss заменить на F.binary_cross_entropy_with_logits, т.к. выход дискриминатора без функции активации.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/somuchkindof2ch/artgan/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>
